### PR TITLE
added Cart model, added payment info to Order model and removed unnec…

### PR DESCRIPTION
**Pull request by wulalasuperdope**
…essary comments from all models

### CODE REVIEW

File: server/db/models/cart.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const Cart = db.define('cart', {
+    id: {
+        primaryKey: true,
+        type: UUID,
+        defaultValue: UUIDV4
+    },
+    productQuantity: {
+        type: INTEGER,
+        allowNull: false,
+        defaultValue: 0,
```
Comment (nries1) [inline server/db/models/cart.js]: Do we want to make default value 1?

File: app/components/root.js [hunk 2]
```diff
@@ -44,8 +44,7 @@ class Root extends React.Component {
             <Route exact path="/products/add" component={AddProductForm} />
             <Route path="/products/:id" component={ProductPage} />
             <Route
-              path="/orders/:orderId/shoppingcart/:userId?" /*component={ShoppingCart} */
-            />
+              path="/shoppingcart/:userId?" component={ShoppingCart} />
```
Comment (nries1) [inline app/components/root.js]: does this render?

**Review by nries1 — DISMISSED** [review]
Very nice!

File: app/components/root.js (hunk 2 above)
Comment (wulalasuperdope) [reply app/components/root.js]: nope. I'll keep the original path and comment out the component.

File: server/db/models/cart.js (hunk 1 above)
Comment (wulalasuperdope) [reply server/db/models/cart.js]: Why 1?

Comment (nries1) [reply server/db/models/cart.js]: I don't think people will add something to cart if they don't want any of it.

**Review by nries1 — DISMISSED** [review]
Nice!

File: server/db/models/cart.js (hunk 1 above)
Comment (wulalasuperdope) [reply server/db/models/cart.js]: Maybe I just remove the default value setting in the model? Seems it's not necessary. The product should be removed from the Cart table if the quantity is zero. Does this sound right?

**Review by nries1 — APPROVED** [review]
Cool that works

**Pull request merged**